src/app/routes/topics/index.js

```diff
@@ -0,0 +1,14 @@
+//imports route and combines them
```

Comment:
need to fix this also before merge

Reply:
Done! Will push with other changes soon. Thanks